lib/matplotlib/tests/test_colors.py

```diff
@@ -1660,3 +1663,25 @@ def test_set_cmap_mismatched_name():
     cmap_returned = plt.get_cmap("wrong-cmap")
     assert cmap_returned == cmap
     assert cmap_returned.name == "wrong-cmap"
+
+
+def test_pathcollection_alpha_none_facecolor():
```

Comment:
Please run the test on `to_rgba_array()` directly, not on `PathCollection`. While you've experienced the problem in `PatchCollection`, the actual fault and behavior change is in that function. It's better to keep the scope of unit tests very focussed.

Reply:
Sure, thanks. I've rewritten the unit test to test specifically the changed code in `to_rgba_array`. Please let me know if there are any problems.
Plus, I am seeing building check failures due to subprocess timeouts which does not seem to be related to my code change. How should I fix this?